misc/azure-release.yml

```diff
@@ -172,3 +172,22 @@ stages:
         find $TILEDB_INSTALL || true
       displayName: "List all the files"
       condition: always()
+
+  - job: trigger_fail_build
+    dependsOn: [build2_python]
+    condition: failed()
+    pool: server
+    steps:
+    - task: InvokeRESTAPI@1
+      inputs:
+        connectionType: 'connectedServiceName'
+        connectedServiceName: 'GitHub-TileDB-Py'
+        method: 'POST'
+        body: |
+          {
+          "event_type": "failed"
+          }
+        urlSuffix: 'repos/TileDB-Inc/TileDB-Py/dispatches'
```

Comment:
Can this repo name be conditional so that we don't accidentally trigger notifications when running on a fork in the future?

Reply:
I tried doing this by using the `Build.Repository.Name` variable to see if it matched `TileDB-Inc/TileDB-Py` but haven't been able to get it to work.

Reply:
Does substitution work here? If we can make it point at the originating repository then that solves the issue - allows testing on a fork w/out impacting the main repo.

Reply:
Actually, let's get this merged as-is and we can do that in follow-up.